libr/core/cmd_info.c

```diff
@@ -253,7 +253,7 @@ static int cmd_info(void *data, const char *input) {
 	int is_array = 0;
 	Sdb *db;
 
-	for (i = 0; input[i] && i < 2; i++) {
+	for (i = 0; input[i]; i++) {
```

Comment:
But it seems to work fine... Only the `mode` is different than before and regression test seems ok too. Is there any command that might be influenced?